packages/components/Tabs/src/Tabs.tsx

```diff
@@ -82,10 +86,48 @@ export const Tabs = compose<TabsType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
+    if (Platform.OS == 'windows') {
+      const pressable = useAsPressable({
+        ...rest,
+      });
+
+      const onKeyDown = (ev: any) => {
+        if (ev.nativeEvent.key === 'ArrowRight' || ev.nativeEvent.key === 'ArrowLeft') {
+          const length = state.context.enabledKeys.length;
+          const currTabItemIndex = state.context.enabledKeys.findIndex(x => x == state.context.selectedKey)
+          let newCurrTabItemIndex;
+          if (ev.nativeEvent.key === 'ArrowRight') {
+            if (isCircularNavigation || !(currTabItemIndex + 1 == length)) {
```

Comment:
Isn't the only case you need to handle `if (isCircularNavigation && currTabItemIndex+1==length)`?

Keyboarding for all other scenarios should already be handled by the underlying platform, correct? I would think keyboarding between the Tab Items would work already, just not the circularNav feature